config/feature.php

```diff
@@ -1,10 +1,9 @@
 <?php
 
 return [
-
-    /*
-     * If true, player-facing beaten games UI/UX is enabled.
-     */
-    'beat' => env('FEATURE_BEAT', false),
-
+/**
+ * EXAMPLE:
+ * // If true, player-facing beaten games UI/UX is enabled.
+ * 'beat' => env('FEATURE_BEAT', false),
+ */
```

Comment:
// 'example' => env('FEATURE_EXAMPLE', false),

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/commit/e57d45732f1ccd3c1694bb9eb63b62d26222a0f6.